include/aws/iotdevice/private/secure_tunneling_operations.h

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#ifndef AWS_IOTDEVICE_SECURE_TUNNELING_OPERATION_H
+#define AWS_IOTDEVICE_SECURE_TUNNELING_OPERATION_H
+
+#include <aws/common/byte_buf.h>
+#include <aws/common/linked_list.h>
+#include <aws/common/ref_count.h>
+#include <aws/iotdevice/private/serializer.h>
+#include <aws/iotdevice/secure_tunneling.h>
+
+/*********************************************************************************************************************
+ * Operations
+ ********************************************************************************************************************/
+
+struct aws_secure_tunnel_operation;
+
+enum aws_secure_tunnel_operation_type {
+    AWS_STOT_NONE,
+    AWS_STOT_PING,
+    AWS_STOT_DATA,
```

Comment:
Yes, I think so but I used DATA to keep in line with the types of messages being used by the protocol in case they add something that'll make MESSAGE no longer make sense. This is for us though so I'm fine changing this to MESSAGE